components/roadmap-grid/grid-row.tsx

```diff
@@ -85,20 +84,22 @@ export function GridRow({
       style={{
         gridColumnStart: `span ${numGridCols / numHeaderItems}`,
         gridColumnEnd: `${closestDateIdx}`,
-        background: `linear-gradient(90deg, rgba(125, 224, 135, 0.6) ${parseInt(
-          milestone.completion_rate.toString(),
-        )}%, white 0%, white ${100 - parseInt(milestone.completion_rate.toString())}%)`,
+        background: '#ffffff',
       }}
       className={`${styles.item} ${styles.issueItem} ${clickable && styles.wrapperLink} js-milestoneCard ${clickable && className}`}
-
     >
-      <Flex direction={{ base:"column", md:"column", lg:"row" }} justify="space-between" position="relative">
-        <Flex direction="column" maxW={{ base: "100%", sm: "100%", md:"100%", lg:"85%" }}>
-          <Text as="b" className={styles.milestoneTitleWrapper}>{milestone.title.value}</Text>
-          <p className={styles.milestoneDate}>{milestone.due_date.value}</p>
+      <Flex direction={{ base:"column" }} justify="space-between" position="relative" w="100%">
+        <Flex direction="column" w="100%">
+          <Text as="b" className={`${styles.milestoneTitleWrapper} ${clickable && styles.milestoneTitleWrapperLink}`}>{milestone.title.value}</Text>
+          <Flex h='8px' w="100%" borderRadius="20px" bgColor="#F1F4F8">
+            <Box w={`${parseInt(milestone.completion_rate.value.toString())}%`} h="100%" borderRadius="20px" bg="#7DE087" />
```

Comment:
I see that this is existing code but I'm confused why are we converting a number to a string to an int to a string? 

```
w={`${parseInt(milestone.completion_rate.value.toString())}%`}
```

If there is some need to truncate, we should use https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_objects/Number/toPrecision where we set `completion_rate`. Could you address that @djmcquillan ?